tests/tck/features/delete/DeleteTag.feature

```diff
@@ -0,0 +1,287 @@
+# Copyright (c) 2021 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+Feature: Delete string vid of tag
+
+  Scenario: delete string vid one vertex one tag
+    Given an empty graph
+    And load "nba" csv data to a new space
+    # before delete tag
+    When executing query:
+      """
+      FETCH PROP ON player "Tim Duncan" YIELD player.name, player.age
+      """
+    Then the result should be, in any order:
+      | VertexID     | player.name  | player.age |
+      | "Tim Duncan" | "Tim Duncan" | 42         |
+    When executing query:
+      """
+      FETCH PROP ON bachelor "Tim Duncan" YIELD bachelor.name, bachelor.speciality
+      """
+    Then the result should be, in any order:
+      | VertexID     | bachelor.name | bachelor.speciality |
+      | "Tim Duncan" | "Tim Duncan"  | "psychology"        |
+    When executing query:
+      """
+      LOOKUP ON player WHERE player.name == "Tim Duncan"
+      """
+    Then the result should be, in any order:
+      | VertexID     |
+      | "Tim Duncan" |
+    # delete one tag
+    When executing query:
+      """
+      DELETE TAG player FROm "Tim Duncan";
+      """
+    Then the execution should be successful
+    # after delete tag
+    When executing query:
+      """
+      FETCH PROP ON player "Tim Duncan" YIELD player.name, player.age
+      """
+    Then the result should be, in any order:
+      | VertexID | player.name | player.age |
+    When executing query:
+      """
+      FETCH PROP ON bachelor "Tim Duncan" YIELD bachelor.name, bachelor.speciality
+      """
+    Then the result should be, in any order:
+      | VertexID     | bachelor.name | bachelor.speciality |
+      | "Tim Duncan" | "Tim Duncan"  | "psychology"        |
+    When executing query:
+      """
+      LOOKUP ON player WHERE player.name == "Tim Duncan"
+      """
+    Then the result should be, in any order:
+      | VertexID |
```

Comment:
Then drop the used space at the end of the scenario.

Reply:
done